app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,23 @@
   </head>
   <body>
     <header>
+      <% if flash.count > 0 %>
+        <section class="flash">
+          <div class="alert alert-danger">
```

Comment:
will need to refactor for success messages

Reply:
refactored!